internal/coreinternal/processor/filterconfig/config.go

```diff
@@ -120,6 +121,11 @@ type MatchProperties struct {
 	Libraries []InstrumentationLibrary `mapstructure:"libraries"`
 }
 
+type DurationProperties struct {
+	Operator string `mapstructure:"operator"`
+	Duration int64  `mapstructure:"duration"`
```

Comment:
I think it would be much less error-prone to use `time.Duration` as the type here